source/wp-content/themes/wporg-news-2021/functions.php

```diff
@@ -9,6 +9,13 @@
 add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\enqueue_assets' );
 add_action( 'admin_init', __NAMESPACE__ . '\editor_styles' );
 
+
+define( 'REPO_TOOLS_DIR', dirname( ABSPATH ) . '/vendor/wporg/wporg-repo-tools' );
```

Comment:
this might need a different path for `wp-env` vs my local nginx env

production will definitely need a different path, since we'll have 1 checkout of `repo-tools`, instead of each theme having their own